ovos_tts_plugin_server/__init__.py

```diff
@@ -78,17 +76,15 @@ def _write_audio_file(self, wav_file: str, data: bytes) -> None:
 
     def _fetch_audio_data(self, params: dict, sentence: str, servers: list) -> bytes:
         """Get audio bytes from servers."""
-        random.shuffle(servers)  # Spread the load among all public servers
         for url in servers:
             try:
                 if self.v2:
                     url = f"{url}/v2/synthesize"
                     params["utterance"] = sentence
                 else:
                     url = f"{url}/synthesize/{sentence}"
-                r: requests.Response = requests.get(
-                    url=url, params=params, verify=self.verify_ssl, timeout=self.tts_timeout
-                )
+                self.log.debug(f"Chosen TTS server {url}")
+                r: requests.Response = requests.get(url=url, params=params, verify=self.verify_ssl, timeout=30)
```

Comment:
This reverts the timeout from the dev branch, need to change it back from hardcoded value